### DISCUSSION TITLE
Github Project 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#48] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/84510455/220045137-1493d3ca-e9cc-4625-9c04-69f29a26bd92.png)

Project에서 In Progress에서 Done으로 옮겼을 때 자동으로 issue가 closed되게 할 수 있나요?

**Comment by jongfeel** [comment, marked as answer]
반대는 자동으로 해주는데
질문한 내용은 안되는 걸로 알고 있습니다.

project 화면에서 오른쪽 위에 버튼에 workflow가 있습니다.
<img width="254" alt="image" src="https://user-images.githubusercontent.com/17442457/220313476-71ae59f8-1cf0-4dc1-a749-fbaec8b195e5.png">

그러면 기본 workflow가 어떻게 세팅되어 있는지 알려주는데
여기서 초록색 불이 켜지지 않은 worlflow를 선택하고 옵션을 바꿔주면 자동화가 어느 정도 이루어집니다.
<img width="217" alt="image" src="https://user-images.githubusercontent.com/17442457/220313805-586a6e72-97b7-47c3-9529-a444651563c0.png">

**Reply by fkdl0048** [reply]
![image](https://user-images.githubusercontent.com/84510455/220314772-51da1acb-c0ec-403d-9e4a-b3cd2a7a1944.png)

조금 설정하니까 추가할 때 정말 편하네요 하나씩 no state에서 옮겼는데..ㅠ  

조금 더 좋아지면 repo별로 추가될 때 상태를 자동으로 분류되면 좋겠네요..!
